ansible/roles/stack-sunbird/templates/sunbird_knowledge-mw-service.env

```diff
@@ -21,7 +21,7 @@ sunbird_content_repo_api_base_url={{sunbird_content_repo_api_base_url}}
 sunbird_content_repo_api_key={{sunbird_content_repo_api_key}}
 sunbird_search_service_api_base_url={{knowledge_mw_sunbird_search_service_api_base_url | default('http://search-service:9000')}}
 sunbird_search_service_api_key={{sunbird_search_service_api_key}}
-sunbird_dial_repo_api_base_url="http://dial-service.{{namespace}}.svc.cluster.local:9000"
+sunbird_dial_repo_api_base_url={{sunbird_dial_repo_api_base_url}}
```

Comment:
Please change this to 
`sunbird_dial_repo_api_base_url=http://dial-service.{{namespace}}.svc.cluster.local:9000`

Reply:
done